tests/fixtures.py

```diff
@@ -775,7 +775,7 @@ def parametrize(params, values, **kwargs):
         ("kernel32", "file", capa.features.file.Export("lstrlenW"), True),
         ("kernel32", "file", capa.features.file.Export("nope"), False),
         # forwarded export
-        ("ea2876", "file", capa.features.file.Export("vresion.GetFileVersionInfoA"), True),
```

Comment:
this is not a typo, please revert this one line

Reply:
@RainRat bump